docs/pages/integration-test/masonry.js

```diff
@@ -7,6 +7,12 @@ import { generateExampleItems } from '../../integration-test-helpers/masonry/ite
 
 const measurementStore = Masonry.createMeasurementStore();
 
+// This is the counterpart to `normalizeValue` in `playwright/masonry/utils/getServerURL.mjs`
+// This will likely need to be updated in the near future
+function booleanize(value: '1' | '0'): boolean {
```

Comment:
`normalizeValue` can use more than just `'1' | '0'`for query param values, but I haven't come across that yet — I'll update this when I do